### ISSUE TITLE
Running with one method only

### ISSUE THREAD

**Issue by axelay** [Braveno/cosigner#1]
I am trying to run a single node but I do not care much for anything else than Bitcoin at present.
After setting the properties file, I fire ```java -jar cosigner-core-0.0.1-SNAPSHOT.jar server core.yml``` but it fails with the following:
```
Exception in thread "main" java.lang.reflect.UndeclaredThrowableException
        at com.sun.proxy.$Proxy34.eth_getTransactionCount(Unknown Source)
        at io.emax.cosigner.fiat.FiatWallet.getSigString(FiatWallet.java:404)
        at io.emax.cosigner.fiat.FiatWallet.getSigString(FiatWallet.java:358)
        at io.emax.cosigner.fiat.FiatWallet.signTransaction(FiatWallet.java:351)
        at io.emax.cosigner.fiat.FiatWallet.signTransaction(FiatWallet.java:313)
        at io.emax.cosigner.fiat.FiatWallet.setupFiatContract(FiatWallet.java:114)
        at io.emax.cosigner.fiat.FiatWallet.<init>(FiatWallet.java:62)
        at io.emax.cosigner.core.CosignerApplication.run(CosignerApplication.java:89)
        at io.emax.cosigner.core.CosignerApplication.run(CosignerApplication.java:26)
        at io.dropwizard.cli.EnvironmentCommand.run(EnvironmentCommand.java:40)
        at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:77)
        at io.dropwizard.cli.Cli.run(Cli.java:70)
        at io.dropwizard.Application.run(Application.java:80)
        at io.emax.cosigner.core.CosignerApplication.main(CosignerApplication.java:59)
Caused by: java.net.ConnectException: Connection refused
        at java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:589)
        at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
        at sun.net.www.http.HttpClient.New(HttpClient.java:308)
        at sun.net.www.http.HttpClient.New(HttpClient.java:326)
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1148)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
        at com.googlecode.jsonrpc4j.JsonRpcHttpClient.invoke(JsonRpcHttpClient.java:146)
        at com.googlecode.jsonrpc4j.ProxyUtil$3.invoke(ProxyUtil.java:242)
        ... 14 more
```
Which I assume is due to the fact that Ethereum has not been properly setup. Anyway to only run a node on a single client?

**Comment by Dorgky** [comment]
In core.yml change ```enabledCurrencies: []``` to ```enabledCurrencies: [BTC]```.

**Comment by axelay** [comment]
Sorry nope - same error no matter what I put in

**Comment by Dorgky** [comment]
The library was trying to create the contract but not handling it when it failed. Fixed with: #2
